spec/outputs/loggly_spec.rb

```diff
@@ -60,9 +56,9 @@ def logger_for(plugin)
       output.receive(event)
     end
 
-    it 'should default tag to logstash if interpolated field for tag does not exist' do
+    it 'should send no tag to logstash if interpolated field for tag does not exist' do
```

Comment:
Initial wording (before your contribution) was a bit wonky. Could you remove the inaccurate "to logstash", and perhaps have this instead?

`it 'should have no tag if the interpolated field for the tag does not exist' do`